Gems/Multiplayer/Code/Include/Multiplayer/Physics/PhysicsUtils.h

```diff
@@ -1,33 +0,0 @@
-/*
```

Comment:
Have concerns about the removal of this code. @kberg-amzn can you confirm we have no future needs for this code?

Reply:
Similar to above, these functions are not implemented

Reply:
If it really is unused then delete away! Probably good to get Sergey to chime in here though.

Reply:
In this particular case, PhysicsUtils.cpp is also being removed, it was not part of the build (was not included by any cmake file). That means that the functions were not implemented, therefore unused.

Reply:
For a moment I thought we use those utils in MultiplayerSample's weapon system, but actually there's a different set of utils in there. So should be safe to remove.
We can then port the utils that are actually used to be in the engine rather than MP sample